metabase/metabase
# Fix Input and dropdown fields have different styles

## Changes

[Visual polish](https://www.notion.so/metabase/Viz-Pod-Rest-of-the-Cycle-Outlook-c0456a67b5754647937bdc1def78e04c#02efa29f7ebc47d0be4f2bd3c34046d0) of the settings changes:

Unify inputs and select components UI:

<video src="https://user-images.githubusercontent.com/14301985/202579201-17c1da4c-7cc6-4eb6-8f71-9d4315402455.mov" />

## How to verify

- Create a question and open a formatting settings popup
- Ensure inputs and dropdowns have similar appearance
